client/components/jetpack/fix-all-threats-dialog-new/index.tsx

```diff
@@ -40,7 +40,7 @@ const FixAllThreatsDialog = ( { onConfirmation, onCloseDialog, showDialog, threa
 
 	const buttons = useMemo(
 		() => [
-			<Button className="fix-all-threats-dialog-new__btn" onClick={ onCloseDialog }>
+			<Button className="fix-all-threats-dialog-new__btn" onClick={ () => onCloseDialog() }>
```

Comment:
Searching for all instances of `onCloseDialog=` shows that there are no instances which use the argument.